docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+
+services:
+  home-assistant:
+    image: homeassistant/home-assistant
+    container_name: home-assistant
+    restart: unless-stopped
+    network_mode: host
+    volumes:
+      - ~/.homeassistant:/config
```

Comment:
Should we be using `~/.homeassistant` as the configuration folder? I mean on most graphical OSes that folder is pretty hard to find from a file explorer.

With docker compose, we have the benefit is that the docker-compose file is at a fixed location contrary to the direct python package. Most docker compose projects I've used store the configuration folders relative to the location of the `docker-compose.yaml` file.

Reply:
My reason behind this is that it is the default location that hass uses when running it via python so I thought keeping it the same would be better.

Also, your suggestion would conflict with those developing on hass. Changing this to `~/homeassistant` would work but imo it clutters up my home dir. The default dir works for me

Reply:
My previous comment was about using the `./homeassistant` folder in the directory where the docker-compose file sits, not `~/homeassistant`.

That being said, `./homeassistant` is not a good folder for development, I agree. The default for development is `./config` (see `.gitignore`).

Ultimately I don't mind if it's in `~/.homeassistant` or somewhere else. I think `~/.homeassistant` is also fine.